File: ETO-parent/ETO-sellergoods-interface/src/main/java/com/ETO/sellergoods/service/BrandService.java
package com.ETO.sellergoods.service;
/*
 * 品牌接口
 * @author DongZhibin
 * */

import java.util.List;
import java.util.Map;

import com.ETO.proj.TbBrand;

import entity.PageResult;
import entity.Result;

public interface BrandService {
	public List<TbBrand> findAll();
	public PageResult findPage(int pageIndex,int pageSize);
	public void add(TbBrand brand);
	public TbBrand findOne(long id);
	public void updateBrand(TbBrand brand);
	public void deleteBrand(long [] ids);
	public PageResult findPage(TbBrand brand,int pageIndex,int pageSize);
	public List<Map> selectOptionList();
}

File: ETO-parent/ETO-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function($http){
		this.findAll = function(){
			return $http.get('../brand/findAll.do');
		}
		this.addBrand = function(brand){
			return $http.post('../brand/addBrand.do',brand);
		}
		this.updateBrand = function(brand){
			return $http.post('../brand/updateBrand.do',brand);
		}
		this.findOne = function(id){
			return $http.get('../brand/findOne.do?id=' + id);
		}
		this.deleteBrand = function(ids){
			return $http.get('../brand/deleteBrand.do?ids=' + ids);
		}
		this.searchBrand = function(pageIndex,pageSize,brandSearch){
			return $http.post('../brand/searchBrand.do?pageIndex='+pageIndex+'&pageSize='+pageSize,brandSearch);
		}
		this.selectOptionList = function(){
			return $http.post('../brand/selectOptionList.do');
		}
});